index.html

```diff
@@ -8,7 +8,7 @@
 </head>
 
 <body>
-
+  <h1>Welcome to Potionfy, Enjoy your stay.</h1>
```

Comment:
Could you remove the 'Enjoy your stay' string and make the 'Welcome to Potionfy' red?

Reply:
Sure, can do!